filters/ema.cpp

```diff
@@ -0,0 +1,25 @@
+#include "filters/ema.hpp"
+
+namespace mesos {
+namespace serenity {
+
+IpcEMAFilter::~IpcEMAFilter() {}
+
+
+Try<Nothing> IpcEMAFilter::consume(ResourceUsage_Executor& in)
```

Comment:
We should consume whole ResourceUsage proto and put "MA" values in place of "non-averaged" values. Keep the pipeline running.